doc/api/test.md

```diff
@@ -3970,6 +3970,14 @@ The absolute path of the test file that created the current suite. If a test
 file imports additional modules that generate suites, the imported suites will
 return the path of the root test file.
 
+### `context.fullName`
+
+<!-- YAML
+added: v22.3.0
```

Comment:
I wonder if this is correct or not. 

Here is the commit adding this: https://github.com/nodejs/node/commit/90e81aaad7198e1bd6536b5b33b5c439786a8595

And PR url: https://github.com/nodejs/node/pull/53169

I think it was backported as far back as v20.16.0 so would that be the more accurate version for this?